### ISSUE TITLE
Make the notify update optional

### ISSUE THREAD

**Issue by nikaiw** [ashirt-ops/aterm#104]
When being run aterm is checking for updates. However it might not be practical when running in a disconnected environement 

Can you make the following call optional e.g from a configuration value ?: 
https://github.com/ashirt-ops/aterm/blob/54a919d8d70dcce77775863de4a32b82dac74245/cmd/aterm/climain.go#L56

**Comment by jrozner** [comment]
Ya, this is definitely something that we can make optional. Is this something that you're ok with being controlled from the config file and being default on but controllable via the setup routine?

**Comment by nikaiw** [comment]
Yeah, I think it would be fine to have it on by default as long as it is at least controllable either via a config file or a command line argument

**Comment by jrozner** [comment]
This was fixed as part of #159
